### ISSUE TITLE
Extend plugin in order to overwrite the place function

### ISSUE THREAD

**Issue by stephenbe** [uxsolutions/bootstrap-datepicker#2131]
Anyway possible to extend the datepicker plugin to overwrite the place function?
https://github.com/uxsolutions/bootstrap-datepicker/blob/master/dist/js/bootstrap-datepicker.js#L650

I want to be able to change the value of var left wihtout changing the plugin directly.
https://github.com/uxsolutions/bootstrap-datepicker/blob/master/dist/js/bootstrap-datepicker.js#L670

Any help on this would be appreciated

**Comment by stephenbe** [comment]
I figured it out : 
```
 $.fn.datepicker.Constructor.prototype.place = function() {
   console.log("overwritten");
 }
```

**Comment by Azaret** [comment]
If I may, why do you want to override the `left` variable?

**Comment by stephenbe** [comment]
I wanted to place the calendar in a different spot then the default : 
![image](https://cloud.githubusercontent.com/assets/3018441/23261272/848e2484-f9a3-11e6-9df7-3e08527aa7ed.png)

Correct me if I am wrong, but I don't think there is any other way to change the placement of the calendar.

**Comment by Azaret** [comment]
This is the [orientation](https://bootstrap-datepicker.readthedocs.io/en/stable/options.html#orientation) option maybe.

**Comment by stephenbe** [comment]
@Azaret I agree 100%, we extended because we wanted to work when you click on the button making it so that the popup is aligned with that button and not the input.

I hope that clarifies me wanting to extend the plugin
